script/script.js

```diff
@@ -396,19 +399,73 @@ emailInput.addEventListener('input', (event) => {
   event.preventDefault();
 });
 
+function storageAvailable(type) {
+  let storage;
+  try {
+    storage = window[type];
+    const x = '__storage_test__';
+    storage.setItem(x, x);
+    storage.removeItem(x);
+    return true;
+  } catch (e) {
+    return e instanceof DOMException && (
+      // everything except Firefox
+      e.code === 22
+      // Firefox
+      || e.code === 1014
+      // test name field too, because code might not be present
+      // everything except Firefox
+      || e.name === 'QuotaExceededError'
+      // Firefox
+      || e.name === 'NS_ERROR_DOM_QUOTA_REACHED')
+      // acknowledge QuotaExceededError only if there's something already stored
+      && (storage && storage.length !== 0);
+  }
+}
+
+function populateLocalStorage() {
+  if (nameInput.value !== null) {
+    localStorageDataObject.name = nameInput.value;
+  }
+  if (emailInput.value !== null) {
+    localStorageDataObject.email = emailInput.value;
+  }
+  if (messageInput.value !== null) {
+    localStorageDataObject.message = messageInput.value;
+  }
+  const localStorageJSON = JSON.stringify(localStorageDataObject);
+  if (storageAvailable('localStorage')) {
+    localStorage.setItem('formData', localStorageJSON);
+  }
+}
```

Comment:
- [ ] Hi please you are required to store the form data in local storage but currently, your function is not working please kindly fix it
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/37901469/195977119-47890c74-8508-4b65-be9a-5353e1feda5b.png">